### DISCUSSION TITLE
Primary concentration in PorousFlowAqueousPreDisChemistry

### DISCUSSION THREAD

**Discussion by steam-simulator** [idaholab/moose#26168] [Q&A Modules: Porous Flow] [answered]
Hello, 
It seems `PorousFlowAqueousPreDisChemistry` treats/uses the mass fractions of the variables representing the chemical species as the primary concentrations . Could someone please confirm this for me? Thanks.

**Comment by cpgr** [comment, marked as answer]
Yep

**Reply by steam-simulator** [reply]
ok. Last question. so I'm assuming this concentration is in mol/m3 (i.e., molarity). correct?

**Reply by cpgr** [reply]
I think it is molarity, but it should be listed here

https://mooseframework.inl.gov/modules/porous_flow/nomenclature.html
